config.lua

```diff
@@ -0,0 +1,251 @@
+lvim.format_on_save = true
```

Comment:
@kylo252 So no `config.lua` in `master` branch and have links in `README.md` for branches with their respective lang/ide configuration... 😕 ( I think the original idea was to have examples configuration for few common languages and plugins etc.. )

Reply:
To clarify, I'm talking about linking to this file instead: https://github.com/LunarVim/LunarVim/blob/d3ade21d85084634517848a7fd3582bbf2d6c278/utils/installer/config.example.lua

We should ideally have a single "common" example `config.lua`, that's always kept up to date with any changes/recommendations. We can always update that file, if you have any concrete ideas, see https://github.com/LunarVim/LunarVim/pull/3519.

Reply:
Sorry for the late response... I've added all the changes to #56 since after removing `config.lua` from this PR it wouldn't be addressing #1 anymore.